docs/rules/no-restricted-imports.md

```diff
@@ -35,6 +35,28 @@ When using the object form, you can also specify an array of gitignore-style pat
 }]
 ```
 
+You may also specify a custom message for any paths you want to restrict as follows:
+
+```json
+"no-restricted-imports": ["error", [{
+  "name": "import-foo",
+  "message": "Please use import-bar instead."
+}]]
+```
+
+or like this:
+
+```json
+"no-restricted-imports": ["error", {
+  "paths": [{
+    "name": "import-foo",
+    "message": "Please use import-bar instead."
+  }]
+}]
+```
+
+The custom message will be appends to the default error message. Please note that you may not specify custom error messages for restricted patterns as a particular import may match more than one pattern.
```

Comment:
I think this should say "The custom message will be appended to the default error message."